src/api/app/components/action_seen_by_user_component.html.haml

```diff
@@ -0,0 +1,19 @@
+- if @render_only
+  - if seen_by_user
+    %i.fa-solid.fa-square-check
+  - else
+    %i.fa-regular.fa-square
```

Comment:
This could be extracted to a method in the component, especially since the exact same code is present on lines 15-18.